Tools/clinic/clinic.py

```diff
@@ -68,7 +69,7 @@ def __repr__(self):
 sig_end_marker = '--'
 
 
-_text_accumulator_nt = collections.namedtuple("_text_accumulator", "text append output")
+_text_accumulator_nt = collections.namedtuple("_text_accumulator_nt", "text append output")
```

Comment:
Consider changing this to a `typing.NamedTuple` at some point (doesn't need to be done now). It'll give the type checker much more information to work with.